### ISSUE TITLE
ES verification unavailable for two verifiers (probably an IE issue)

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#918]
Two verifiers have reported that the ES verification page is refusing to load - they just get the egg-timer. At least one of the two uses Internet Explorer, so that may be the problem. I've just tried getting on to the the ES verification page myself using IE11 and I also get stuck on the egg-timer.

It would be good to retain IE compatability for as long as we can.

**Comment by burkmarr** [comment]
@johnvanbreda - I replicated this on IE11. Your huch was right - some JS not compatible with IE11. It's in the applyQueryStringFiltersToRequest function in idc.core.js. It's using URLSearchParams which IE11 doesn't know about - https://caniuse.com/?search=URLSearchParams.

**Comment by BirenRathod** [comment]
@kitenetter 
Is it possible to ask these two verifiers to not to use IE11? As IE is stop getting support from Microsoft, implementing JS is not a possible solution.

**Comment by kitenetter** [comment]
@BirenRathod yes, it is possible, and has been done, but some users are reluctant to change! I will keep trying to persuade.

**Comment by DavidHepper** [comment]
I agree with Biren.

**Comment by johnvanbreda** [comment]
I'll add a polyfill to the code for this specific case but we should remove it for the sake of tidiness once the final few verifiers move over.

**Comment by johnvanbreda** [comment]
Now done - thanks @burkmarr for the investigation.

**Comment by kitenetter** [comment]
We have at least one very active verifier who is absolutely wedded to IE. I think what we need to do is to make a decision as to whether we can continue to support IE - if not that is entirely understandable, but it would be preferable to announce in advance that we are going to stop doing so and give people a few weeks or months of warning that they will have to change their browser.

**Comment by kitenetter** [comment]
@BirenRathod I entirely agree with you but we haven't (as far as I'm aware) yet made this clear to our verifiers and other users. I will discuss with @DavidRoy how best to commmunicate this.

**Comment by johnvanbreda** [comment]
Although Microsoft is dropping support for IE11, that's not happened yet so we probably should not drop it just yet. In some environments such as council offices (where LRCs might be), it can be difficult to upgrade away from old machines, operating systems and software. At the moment, IE11 support is not causing huge problems, but perhaps it is a good time to let verifiers know that it won't be supported for ever.

**Comment by BirenRathod** [comment]
If users can't able to drop IE11 in that case we should ask them to go with alternatives like Chrome or Firefox.

We could do this via Forums and put some message on new verify page.
